src/components/Book.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ title, author }) => (
+  <>
+    <h3>{title}</h3>
+    <span>{author}</span>
+  </>
+);
+
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly ensure your book has a `Remove` button to meet the project requirement below:
:point_down: 
![req](https://user-images.githubusercontent.com/59397159/173070824-c26f2c5f-2b14-488a-a257-f31d3136d352.png)